src/main/java/com.gigssandbox/UserInteraction.java

```diff
@@ -0,0 +1,97 @@
+package com.gigssandbox;
+
+import com.gigssandbox.entities.Band;
+import com.gigssandbox.entities.Gig;
+import com.gigssandbox.exceptions.AddingToDatabaseException;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Properties;
+
+class UserInteraction {
+
+    private GigsRepository repository = new GigsRepository();
+    private ConsoleHelper helper;
+    private Properties properties;
+
+    UserInteraction(Properties properties, ConsoleHelper helper) {
+        this.properties = properties;
+        this.helper = helper;
+    }
+
+    void processAction(int actionCode) {
+        switch (actionCode) {
+            case 1:
```

Comment:
Such hardcoded commands are too hard to read and understand. Its better to use Strings or Enum (we may try this)